tracing-subscriber/src/fmt/mod.rs

```diff
@@ -7,18 +7,96 @@
 //! implementation of the [`Subscriber`] trait that records `tracing`'s `Event`s
 //! and `Span`s by formatting them as text and logging them to stdout.
 //!
+//! ## Usage
+//!
+//! First, add this to your `Cargo.toml` file:
+//!
+//! ```toml
+//! [dependencies]
+//! tracing-subscriber = "0.1"
+//! ```
+//!
+//! Add the following to your executable to initialize the default subscriber:
+//! ```rust
+//! use tracing_subscriber;
+//!
+//! tracing_subscriber::fmt::init();
+//! ```
+//!
+//! ## Filtering Events with Environment Variables
+//!
+//! The default subscriber enables you to filter events at runtime using environment variables (using the [`EnvFilter`]).
```

Comment:
Nit: can we add something like 
```suggestion
//! The default subscriber installed by `init` enables you to filter events at runtime using environment variables (using the [`EnvFilter`]).
```